CHANGELOG.md

```diff
@@ -95,6 +95,10 @@
 
 #### Bug Fixes
 
+* Fix `explicit_type_interface` when used in statements.  
```

Comment:
this is in the wrong section, can you update this before merging?

Reply:
Fixed